views/base.html

```diff
@@ -4,6 +4,7 @@
     <title>{{title or "PyJudge"}}</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="/static/css/bootstrap.min.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700,900" rel="stylesheet">
     <script src="/static/js/jquery.min.js"></script>
```

Comment:
can we not change base for now? fonts are not so important and I don't think it will be good to use one in base

Reply:
yes

Reply:
resolving this

Reply:
resolve.